src/components/organizations/show/index.js

```diff
@@ -1,21 +1,24 @@
 import React, {Component, PropTypes} from 'react'
-import { connect } from 'react-redux'
+import {connect} from 'react-redux'
 
 import ReactDOM from 'react-dom'
 import Icon from 'react-fa'
 
 import {SpaceCard, NewSpaceCard} from 'gComponents/spaces/cards'
+
```

Comment:
I don't think you need some of these imports or connected selectors anymore on the org show page, right?